afterpay/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,6 @@
     <string name="load_error_retry">Retry</string>
     <string name="load_error_cancel">Cancel</string>
 
+    <string name="badge_content_description">Afterpay it in 4. Always interest-free.</string>
```

Comment:
Is this for accessibility? I had trouble with the pronunciation of Afterpay and ended up using "after pay". On iOS you can input the IPA pronunciation for something. However this locks it into the American voice it seems and doesn't sound as natural.

I'm not 100% on what we want to use as the content string. Was this specified somewhere or should be raise it with the team?

Reply:
Yeah, this is for accessibility. It reads Afterpay very naturally in the Australian voice but that's a good point, I'll test with other voices.

I'm not 100% sure either and this is basically the placeholder. I went with the phrase that is used almost exclusively when placed next to a badge in the branding guide.

Reply:
Changed to "After pay" to help the screen reader.